packages/okta-angular/test/e2e/harness/src/app/app.module.ts

```diff
@@ -48,7 +48,7 @@ const appRoutes: Routes = [
   },
   {
     path: 'sessionToken-login',
-    component: SessionTokenLogin
+    component: SessionTokenLoginComponent
```

Comment:
This looks like a breaking change?

Reply:
This is used only in our test suite. We aren't exposing any of this via our SDK, so we aren't introducing a breaking change.